package.json

```diff
@@ -25,15 +25,16 @@
   },
   "homepage": "https://github.com/enhance-dev/enhance-morphdom-mixin#readme",
   "devDependencies": {
-    "@enhance/custom-element": "^1.2.3",
+    "@enhance/custom-element": "^1.2.3-next.0",
```

Comment:
This seems to be pointing to an older release. I had to do a:

```bash
npm install "https://github.com/enhance-dev/enhance-custom-element.git#dom-diffing-slots" --save
```

and then I could get the tests to pass.

Reply:
I had reverted to `v1.2.3` in order to use the latest version of custom element when it was released.